apps/utils/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "essential-app-utils"
 description = "Utilities for building Essential applications"
-version = "0.4.0"
+version = "0.5.0"
```

Comment:
Looks like the version on crates.io is already 0.5.0 🤔

Reply:
Yeh I had to manually publish to get this PR to pass